.github/workflows/deploy-changed-cf.yaml

```diff
@@ -78,26 +78,33 @@ jobs:
         name: Validate workflow dispatch
         run: |-
           echo Deploying the master branch via workflow_dispatch is not supported.
-          echo To deploy master, raise and a merge a PR with changes to the
-          echo CF files that needs to be deployed.
+          echo To deploy master branch, raise a PR with changes to the CF files
+          echo that needs to be deployed and merge the PR.
           exit 1
 
-      - id: get-changed-files-on-merge
+      - id: get-changed-files-on-pr-merge
         if: github.event_name == 'push' && github.ref == 'refs/heads/master'
-        name: Get changed files on PR merge
+        name: Get changed files on PR merge to master
         uses: lots0logs/gh-action-get-changed-files@2.1.4
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
 
-      - id: get-changed-files-on-manual-trigger
-        if: github.event_name == 'workflow_dispatch' && github.ref != 'refs/heads/master'
```

Comment:
Removing `github.event_name == 'workflow_dispatch'` allow for linting of changed CF files in branch even if the branch is not triggered